command/show.go

```diff
@@ -2,6 +2,7 @@ package command
 
 import (
 	"fmt"
+	"github.com/hashicorp/terraform/states/statemgr"
```

Comment:
It looks like you're having the same VSCode extension troubles as me here :confounded: ... the addition of new stuff is working but the goimports pushing of non-builtin stuff down into the second list isn't.

I hate nit-picking this stuff but I think we should manually fix this here just so that future edits to this file once the extension is working properly again won't cause this to get reformatted.